man/org.freedesktop.systemd1.xml

```diff
@@ -5810,6 +5818,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2esocket {
 
     <!--property ManagedOOMMemoryPressureLimit is not documented!-->
 
+    <!--property ManagedOOMMemoryPressureDurationUSec is not documented!-->
```

Comment:
can you add a brief paragaprgah about this new prop? for newly added stuff we really try to do better and always have at least *something* from the beginning. It's fine to just say: maps to unit file setting blablah, see that man page for details. Note the unit is USec though.

Reply:
Done in most recent version I'm about to push. Note I didn't add docs for mount/swap/socket. Even though the dbus property is there, its not valid for these unit types